.changeset/wicked-terms-push.md

```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-extension-dsl-persistence': patch
+'@finos/legend-application-studio': patch
+'@finos/legend-graph': patch
+---
```

Comment:
Can you split these changesets please?
Have on changeset for `dsl-persistence` with description 
`Add new assertions types for persistence tests` 
and a different changeset for the other two packages with description `Add extension mechanism for TestAssertion`

Reply:
Sure, changed!